src/simplification/Simplifier.ts

```diff
@@ -0,0 +1,82 @@
+
+import { CommonModel, Schema } from '../models';
+import simplifyProperties from './SimplifyProperties';
+import simplifyEnums from './SimplifyEnums';
+import simplifyTypes from './SimplifyTypes';
+import simplifyItems from './SimplifyItems';
```

Comment:
Maybe you should add those function inside Simplifier as static methods?

Reply:
I would rather have them separated completely tbh instead of everything inside one class 😄

Reply:
Do you use in other place these function? I think you can include them inside this class :) Also you can make something like `private static {staticMethodName}` and then use inside another static methods `this.{staticMethodName}` 😄

Reply:
I mean I use them in tests separately. Do you suggest adding everything from the separate functions and add them as static functions inside the `Simplifier` class? 🤔

Reply:
If it is a problem for you, you can leave it, but functions are only used inside `simplify` function

Reply:
Yes, but IMO separate files for these simplification rules are the best solution. The class file in the end would be way too bloated.

Reply:
OK 😄